docs/source/user_documentation/replica_dot_yaml_file.rst

```diff
@@ -288,6 +389,55 @@ The *specified relation* is ``SNOWSHU_DEVELOPMENT.SOURCE_SYSTEM.ORDER_ITEMS``. W
 - All the records in ``SNOWSHU_DEVELOPMENT.SOURCE_SYSTEM.ORDER_ITEMS`` will be records with an ``order_id`` found in ``SNOWSHU_DEVELOPMENT.SOURCE_SYSTEM.ORDERS``.
 - The records in ``SNOWSHU_DEVELOPMENT.SOURCE_SYSTEM.ORDERS`` *may* be records with an ``id`` not found in ``SNOWSHU_DEVELOPMENT.SOURCE_SYSTEM.ORDER_ITEMS``. 
 
+The example below shows that of polymorphic relationships: 
+
+
+.. code-block:: yaml
+
+   ...
+    - database: SNOWSHU_DEVELOPMENT
+      schema: POLYMORPHIC_DATA
+      relation: PARENT_TABLE_2
+      relationships:
+        polymorphic:
+          - local_attribute: ID
+            database: SNOWSHU_DEVELOPMENT
+            schema: POLYMORPHIC_DATA
+            relation: '(?i)^CHILD_TYPE_[0-9]_ITEMS$'
+            remote_attribute: PARENT_2_ID
+
+
+The *specified relation* is ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.PARENT_TABLE_2`` which relates with a child relations ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.(?i)^CHILD_TYPE_[0-9]_ITEMS$``. When SnowShu builds this replica:
+
+- All the records in ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.PARENT_TABLE_2`` will be records with a ``ID`` found in the ``PARENT_2_ID`` of any tables that match ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.(?i)^CHILD_TYPE_[0-9]_ITEMS$`` (example: ``PARENT_2_ID`` in ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.CHILD_TYPE_2_ITEMS``).
+
+.. code-block:: yaml
+
+   ...
+    - database: SNOWSHU_DEVELOPMENT
+      schema: POLYMORPHIC_DATA
+      relation: PARENT_TABLE
+      relationships:
+        polymorphic:
+          - local_attribute: CHILD_ID
+            local_type_attribute: CHILD_TYPE
+            database: ''
+            schema: ''
+            relation: '(?i)^CHILD_TYPE_[0-9]_ITEMS$'
+            remote_attribute: ID
+            local_type_overrides:
+              - database: SNOWSHU_DEVELOPMENT
+                schema: POLYMORPHIC_DATA
+                relation: CHILD_TYPE_2_ITEMS
+                override_value: type_2
+
+
+The *specified relation* is ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.PARENT_TABLE`` which relates with a child relations ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.(?i)^CHILD_TYPE_[0-9]_ITEMS$``. When SnowShu builds this replica:
+
+- All the records in ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.PARENT_TABLE`` will be records with a ``CHILD_ID`` of ``local_type_attribute``, ``CHILD_TYPE``, found in the ``ID`` of any tables that match ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.(?i)^CHILD_TYPE_[0-9]_ITEMS$`` (example: ``ID`` in ``SNOWSHU_DEVELOPMENT.POLYMORPHIC_DATA.CHILD_TYPE_3_ITEMS``).
```

Comment:
I think this is right, but could probably be clearer. Here's one stab at it that should be correct, but probably could also be phrased better

"All records in the parent table will be records that have a `CHILD_TYPE` (the `local_type_attribute` column) value that matches the child table name (or the `local_type_override` for the child table if used), and a `CHILD_ID` value that matches `ID` value in that matching table."